### ISSUE TITLE
[feature request] README: please add the difference from Nixnote

### ISSUE THREAD

**Issue by vitaly-zdanevich** [d1vanov/quentier#417]

**Comment by d1vanov** [comment]
What could I possibly write about this topic? Quentier is entirely separate from Nixnote, they only share a couple of snippets of code but it's only because the latest maintainer of Nixnote took them from Quentier ) I don't mind it though, both projects are open source.

**Comment by vitaly-zdanevich** [comment]
But why starting a new project instead of forking NixNote?

**Comment by d1vanov** [comment]
I started developing Quentier long before I learned about NixNote's existence. Furthermore, even after I learned of NixNote's existence, I found NixNote design decisions not really aligning with my vision so I just continued to develop Quentier.

**Comment by vitaly-zdanevich** [comment]
From the README:

> At the moment Quentier is in public alpha state and is not yet intended for production usage. Currently it's targeted at software enthusiasts, developers and people who like tinkering with fresh stuff.

Is it still true?

**Comment by d1vanov** [comment]
Yes, it is. There are known problems and limitations with current synchronization logic design and I'm trying to rewrite it into something more robust at the moment. If I ever succeed at that, I might at some point declare that it's no longer alpha but perhaps a public beta version :)

**Comment by boo-yee** [comment]
> What could I possibly write about this topic? Quentier is entirely separate from Nixnote, they only share a couple of snippets of code but it's only because the latest maintainer of Nixnote took them from Quentier ) I don't mind it though, both projects are open source.

Hi, @d1vanov For the sake of searching for the information about rate limit issue, I came to this project issue list and found this page. May I ask whether the latest mainainer of Nixnote who took code from Quentier is me? I have seldom looked at Nixnote code since 2023, and not even login github, so I just cannot remember it. As I remember, I didn't read quentier code too much. If it is me, could you please tell me which part, as I don't want to be a person who signs his name on other people's work, even though it is open source software.

**Comment by d1vanov** [comment]
I meant @robert7 but let me reiterate: it's quite ok to take code from other projects with compatible licenses, that's what open source is about - sharing knowledge as code. I'm not mad about it at all and I don't think that anything needs to be changed in Nixnote2.

If you are curious anyway, the code in question is located here: https://github.com/robert7/nixnote2/tree/master/src/quentier/utility. My original copyright notice is preserved in these files so I have literally nothing to complain about in this case.

**Comment by boo-yee** [comment]
OK. It seems that I have misunderstood that. I thought that someone took your code without copyright notice perserved, and I happened to do some maintain work. I can't remember exactly because there are large amount of code and too long time. Sorry for disturbing you.
